kvakk_git_tools/recommended/gitconfig-dapla-lab

```diff
@@ -0,0 +1,20 @@
+[color]
+	ui = auto
+[core]
+	autocrlf = input
+	editor = nano
+	eol = lf
+[credential]
+	helper = cache --timeout=86400
+[diff "ipynb"]
+	textconv = '/opt/conda/bin/python3' -m nbstripout -t
```

Comment:
this should use global `nbstripout` instead of through python

Reply:
good catch!